packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -157,9 +160,6 @@ export const PaymentForm = ({
         spellCheck={false}
         onValidate={value => {
           let error = null;
-          if (value !== null && !value) {
-            error = 'Please enter your name';
```

Comment:
Removing this means that there is *no* validation error at all on an empty name field - I can submit without entering a name.